src/Http/Routing/test/testassets/Benchmarks/Benchmarks.csproj

```diff
@@ -1,8 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
-
   <PropertyGroup>
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
-    <TargetFramework Condition="'$(BenchmarksTargetFramework)' != ''">$(BenchmarksTargetFramework)</TargetFramework>
     <UseP2PReferences Condition="'$(UseP2PReferences)'=='' AND '$(BenchmarksTargetFramework)'==''">true</UseP2PReferences>
```

Comment:
@sebastienros this is the one remaining mention of `'$(BenchmarksTargetFramework)` in the repo. Should I remove ` AND '$(BenchmarksTargetFramework)'==''` or change the `Condition` to use another property❔

Reply:
Remove

Reply:
Remove the line entirely or just `AND '$(BenchmarksTargetFramework)'==''`❔